components/filter/filter.js

```diff
@@ -617,7 +618,7 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 		return html`
 			<d2l-list-item
 				id="${itemId}"
-				@d2l-list-item-selected="${ifDefined(item.additionalContent ? this._handleListItemSelelcted : undefined)}"
+				@d2l-list-item-selected="${item.additionalContent ? this._handleListItemSelected : undefined}"
```

Comment:
Wrapping this in `ifDefined` does nothing since that's just for attributes. Also fixing typo in the handler name.